worker/uniter/relation/resolver_test.go

```diff
@@ -634,19 +720,13 @@ func (s *relationResolverSuite) TestHookRelationBrokenWhenSuspended(c *gc.C) {
 	relationsResolver := relation.NewRelationResolver(r, nil)
 	op, err := relationsResolver.NextOp(localState, remoteState, &mockOperations{})
 	c.Assert(err, jc.ErrorIsNil)
-	assertNumCalls(c, &numCalls, 11)
+	assertNumCalls(c, &numCalls, 18)
```

Comment:
That's a rather significant bump in the number of API calls. Should we be worried?